pytest/tests/mocknet/mirror.py

```diff
@@ -328,14 +329,21 @@ def stop_traffic_cmd(args, traffic_generator, nodes):
 
 
 def neard_runner_jsonrpc(node, method, params=[]):
-    j = {'method': method, 'params': params, 'id': 'dontcare', 'jsonrpc': '2.0'}
-    r = requests.post(f'http://{node.machine.ip}:3000', json=j, timeout=30)
-    if r.status_code != 200:
-        logger.warning(
-            f'bad response {r.status_code} trying to send {method} JSON RPC to neard runner on {node.instance_name}:\n{r.content}'
+    body = {
+        'method': method,
+        'params': params,
+        'id': 'dontcare',
+        'jsonrpc': '2.0'
+    }
+    body = json.dumps(body)
+    r = run_cmd(node, f'curl localhost:3000 -d \'{body}\'')
```

Comment:
Not sure if that’s important? but here if `method` or `params` contain a quote, then this will fail. Might need to replace all `'` in `body` with `'"'"'` first

Reply:
I would feel slightly better with some escaping provided by a library but as long as this works it's ok.